packages/hoppscotch-sh-admin/src/pages/users/invited.vue

```diff
@@ -16,64 +16,33 @@
           <div v-if="fetching" class="flex justify-center">
             <HoppSmartSpinner />
           </div>
-          <div v-else-if="error || invitedUsers === undefined">
+          <div v-else-if="error">
+            <p class="text-xl">
+              {{ t('users.invite_load_list_error') }}
+            </p>
+          </div>
+          <div
+            v-else-if="invitedUsers === undefined || invitedUsers.length === 0"
+          >
             <p class="text-xl">{{ t('users.no_invite') }}</p>
           </div>
 
-          <table v-else class="w-full text-left">
-            <thead>
-              <tr
-                class="text-secondary border-b border-dividerDark text-sm text-left"
-              >
-                <th class="px-3 pb-3">{{ t('users.admin_id') }}</th>
-                <th class="px-3 pb-3">{{ t('users.admin_email') }}</th>
-                <th class="px-3 pb-3">{{ t('users.invitee_email') }}</th>
-                <th class="px-3 pb-3">{{ t('users.invited_on') }}</th>
-              </tr>
-            </thead>
-            <tbody class="divide-y divide-divider">
-              <tr
-                v-if="invitedUsers.length === 0"
-                class="text-secondaryDark py-4"
-              >
-                <div class="py-6 px-3">{{ t('users.no_invite') }}</div>
-              </tr>
-              <tr
-                v-else
-                v-for="(user, index) in invitedUsers"
-                :key="index"
-                class="text-secondaryDark hover:bg-zinc-800 hover:cursor-pointer rounded-xl"
-              >
-                <td class="py-2 px-3 max-w-30">
-                  <div>
-                    <span class="truncate">
-                      {{ user?.adminUid }}
-                    </span>
-                  </div>
-                </td>
-                <td class="py-2 px-3">
-                  <span class="flex items-center">
-                    {{ user?.adminEmail }}
+          <div v-else>
```

Comment:
We can probably swap the order of these elements so the truthy check comes above, and the empty state goes as a fallback. Also, preferably let's use the same property instead of referring to both `invitedUsers` and `newInvitedUsers` (also a good candidate for renaming).


```vue
<div v-if="newInvitedUsersList?.length">
  <HoppSmartTable ...>
  ...
  </HoppSmartTable>
</div>

<!-- Empty state  -->
<div v-else>
</div>
```